<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainTextures</class>
 <widget class="QDialog" name="TerrainTextures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/resourcen.qrc">
    <normaloff>:/icons/terrain.png</normaloff>:/icons/terrain.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="3" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="default_gloss">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_gloss">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Gloss</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="icons/resourcen.qrc">
       <normaloff>:/icons/gtk-cancel.png</normaloff>:/icons/gtk-cancel.png</iconset>
     </property>
     <property name="shortcut">
      <string>Esc</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="layer_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="icons/resourcen.qrc">
       <normaloff>:/icons/gtk-apply.png</normaloff>:/icons/gtk-apply.png</iconset>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="default_height">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="default_specular">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_height">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_specular">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Specular</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListWidget" name="terrain_textures">
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="blend_size">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_blend_size">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Blend Size</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/resourcen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TerrainTextures</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TerrainTextures</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
